decisions/models/organization.py

```diff
@@ -6,8 +6,16 @@
 from .base import DataModel
 
 
+class OrganizationClass(DataModel):
+    id = models.PositiveIntegerField(primary_key=True)
+    name_fi = models.CharField(max_length=255, null=True)
```

Comment:
This probably warrants a separate PR. The Organization and Post class need translatability too.